public/javascripts/i18n.js

```diff
@@ -12,19 +12,23 @@
 // See tests for specific formatting like numbers and dates.
```

Comment:
Please don't commit this file! This file gets overridden by the original code automatically.